projects/packages/publicize/src/jetpack-social-settings/class-settings.php

```diff
@@ -153,6 +187,7 @@ public function get_settings( $with_available = false ) {
 
 		$settings = array(
 			'socialImageGeneratorSettings' => $this->get_image_generator_settings(),
+			'utmSettings'                  => $this->get_utm_settings(),
```

Comment:
I think this is no longer needed because this `get_settings` method is used only for the initial state. Removing this will also fix the failing PHP unit tests.

Reply:
I'll remove it then I thought we still need it